recipe/migrations/osx_arm64.txt

```diff
@@ -39,6 +39,7 @@ libtool
 autoconf
 automake
 m4
+python-chromedriver-binary
```

Comment:
I'm not sure but isn't this a repackaged binary? If so the bot probably won't help.

Reply:
yes, it's basically a `noarch` that uses itself to download a binary during  the build.

Reply:
I guess one should just add the `osx-arm64` there directly then. Not sure if there is a reason to add it to the bot list.